finance-models-api/src/main/kotlin/com/ajaxproject/financemodelsapi/FinanceModelsApiApplication.kt

```diff
@@ -0,0 +1,11 @@
+package com.ajaxproject.financemodelsapi
+
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.runApplication
+
+@SpringBootApplication
+class FinanceModelsApiApplication
+
+fun main(vararg args: String) {
```

Comment:
why do you need the main function in this subproject?

Reply:
It's unnecessary, I'll delete it